coral/src/app/features/components/AclDetailsModalContent.tsx

```diff
@@ -88,7 +88,16 @@ const DetailsModalContent = ({ aclRequest }: DetailsModalContentProps) => {
         <Flexbox direction={"column"}>
           <Label>Consumer group</Label>
           <dd>
-            {aclType === "CONSUMER" ? consumergroup : <i>Not applicable</i>}
+            {
+              // If consumergroup is "-na-", it means the request was made for an Aiven cluster
+              // Which does not user consumergroup
+              // So we return Not applicable
+              aclType === "CONSUMER" || consumergroup == "-na-" ? (
```

Comment:
not for this PR right now, but for the future we could maybe add a constant like "AIVEN_CLUSTER_IDENTIFIER" (*) for that one, so it's more understandable through the code what happens here and why. 

(*) well, a better name :D that describes the connection better, that's not a good suggestion

Reply:
Hmm. It is for sure data we would like to be able to access easily. the `getClusterInfo` endpoint allows us to know if an environment is an Aiven cluster or not. 

I think we may soon come to a spot where "data that doesn't change very often" (environments, teams maybe?) would be fetched once on first load of the app and then stored in a context, and then never fetched again). We could use `useExtendedEnvironments` to store env data + `aivenCluster` data 🤔 

Then the context could expose an `isAivenCluster` fn that would check if a given env is an aiven cluster against this stored data.

In the context of this component, something like 

```
const { isAivenCluster } = useEnvironmentsData()

//...

 aclType === "CONSUMER" || isAivenCluster(environment) ? ...
```

Reply:
Oh sorry, I was not super clear here. I meant addings something like 
`const VALUE_REPRESENTS_AN_AIVEN_CLUSTER =  "-na-"` 
once and then use that where we currently use `"-na-"`. 

Only for readability of the code. 

Currently we have comments mostly where we use the string "-na-". Which is also fine :D it was more me thinking aloud